app/models/sac_cas/wizards/base.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2025, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module SacCas::Wizards::Base
+  def self.prepended(base)
```

Comment:
Warum nicht als `ActiveSupport::Concern`?

Reply:
`ActiveSupport::Concern` macht einiges was ich in diesem simplen Fall nicht brauche